.rubocop_todo.yml

```diff
@@ -105,76 +57,28 @@ Layout/IndentHeredoc:
 
 # Offense count: 1
 # Cop supports --auto-correct.
-# Configuration parameters: Width, IgnoredPatterns.
-Layout/IndentationWidth:
-  Exclude:
-    - 'features/support/env.rb'
-
-# Offense count: 2
-# Cop supports --auto-correct.
-# Configuration parameters: EnforcedStyle, IndentationWidth.
-# SupportedStyles: aligned, indented
-Layout/MultilineOperationIndentation:
-  Exclude:
-    - 'features/support/env.rb'
-
-# Offense count: 2
-# Cop supports --auto-correct.
-Layout/SpaceAroundKeyword:
-  Exclude:
-    - 'features/step_definitions/cucumber_rails_steps.rb'
-    - 'features/support/env.rb'
-
-# Offense count: 2
-# Cop supports --auto-correct.
 # Configuration parameters: AllowForAlignment.
 Layout/SpaceAroundOperators:
   Exclude:
     - 'dev_tasks/yard.rake'
-    - 'lib/cucumber/rails/database.rb'
-
-# Offense count: 3
-# Cop supports --auto-correct.
-# Configuration parameters: EnforcedStyle, EnforcedStyleForEmptyBraces.
-# SupportedStyles: space, no_space
-# SupportedStylesForEmptyBraces: space, no_space
-Layout/SpaceBeforeBlockBraces:
-  Exclude:
-    - 'lib/cucumber/rails.rb'
-    - 'lib/cucumber/rails/database.rb'
-
-# Offense count: 1
-# Cop supports --auto-correct.
-Layout/SpaceBeforeComma:
-  Exclude:
-    - 'features/step_definitions/cucumber_rails_steps.rb'
 
 # Offense count: 2
 # Cop supports --auto-correct.
 Layout/SpaceInsidePercentLiteralDelimiters:
   Exclude:
     - 'lib/cucumber/rails/database.rb'
 
-# Offense count: 1
-# Cop supports --auto-correct.
-# Configuration parameters: EnforcedStyle.
-# SupportedStyles: space, no_space
-Layout/SpaceInsideStringInterpolation:
-  Exclude:
-    - 'lib/cucumber/rails/database.rb'
-
-# Offense count: 2
-# Cop supports --auto-correct.
-# Configuration parameters: AllowInHeredoc.
-Layout/TrailingWhitespace:
+# Offense count: 37
+Lint/AmbiguousRegexpLiteral:
   Exclude:
     - 'features/step_definitions/cucumber_rails_steps.rb'
-    - 'features/support/env.rb'
+    - 'tmp/aruba/rails-app/features/step_definitions/rest_steps.rb'
+    - 'tmp/aruba/rails-app/features/step_definitions/web_steps.rb'
 
-# Offense count: 10
-Lint/AmbiguousRegexpLiteral:
+# Offense count: 1
+Lint/Debugger:
```

Comment:
We should probably exclude tmp/aruba in .rubocop.yml.

Reply:
I'm tackling that in my 2nd PR. So I'll merge this as is.